core/src/components/card/card.tsx

```diff
@@ -7,7 +7,7 @@ import { generateUniqueId, hasSlot } from '../../utils/utils';
  * @slot thumbnail - Slot for the Card thumbnail.
  * @slot body - Slot for the body section of the Card.
  * @slot body-image - Slot for the body section of the Card, used for image.
- * @slot bottom - Slot for the bottom section of the Card.
+ * @slot actions - Slot for the bottom section of the Card, used for buttons .
```

Comment:
Looks like the "bottom" slot is still used in the stories file for the Card component?
![image](https://github.com/scania-digital-design-system/tegel/assets/48509966/9276b515-9250-4d0d-b268-eaccfaecd006)